### PR TITLE
Enable etcd-wrapper on arm64.

### PR DESCRIPTION
**Pull request by glevand**
Also adds missing ebuild metadata.

Same ebuild structure as https://github.com/coreos/coreos-overlay/pull/2511 (Reenable flannel on arm64).

To work as-is on arm64 the etcd-wrapper needs https://github.com/coreos/etcd/pull/7714 and an arm64 release to quay.io/coreos/etcd.

For testing, an etcd systemd drop-in with something like this should work:

    [Service]

    Environment="ETCD_IMAGE=docker://quay.io/glevand/etcd:v3.1.0-rc.1-arm64"
    Environment="RKT_RUN_ARGS=--uuid-file-save=/var/lib/coreos/etcd-member-wrapper.uuid --insecure-options=image"
    Environment="ETCD_UNSUPPORTED_ARCH=arm64"

### CODE REVIEW

**Comment by glevand** [comment]
@crawford Did you want to rename `etcd-wrapper-0.0.4.ebuild` to `etcd-wrapper-3.0.10.ebuild` so the ebuild package version matches the etcd release?  Then we can use ${PV} in the ebuild, and it makes it easier to know which etcd version the package uses.

**Comment by crawford** [comment]
Seems like a reasonable idea. I'd be fine with it in a separate commit in this PR.

**Comment by glevand** [comment]
Added patch `	etcd-wrapper: Change ebuild version to etcd release`.

**Comment by glevand** [comment]
Rebased to lastest.
Dropped patch `etcd-wrapper: Change ebuild version to etcd release`, which became duplicated in  https://github.com/coreos/coreos-overlay/pull/2549 (app-admin/etcd-wrapper: bump to etcd 3.1.6).

File: coreos-base/coreos/coreos-0.0.1.ebuild [hunk 1]
```diff
@@ -75,9 +75,7 @@ RDEPEND="${RDEPEND}
 	)
 	etcd_protocols_2? (
 		dev-db/etcd:2
-		amd64? (
-			app-admin/etcd-wrapper
-		)
+		app-admin/etcd-wrapper
```
Comment (dm0-) [inline coreos-base/coreos/coreos-0.0.1.ebuild]: The `coreos` ebuild revision number should probably be bumped for this.  (The symlink in the same directory.)  Maybe the `coreos-base/coreos` changes could be in their own commit.

File: app-admin/etcd-wrapper/etcd-wrapper-3.1.6.ebuild [hunk 2]
```diff
@@ -22,8 +22,15 @@ RDEPEND=">=app-emulation/rkt-1.9.1[rkt_stage1_fly]"
 S=${WORKDIR}
 
 src_install() {
+	local tag="v3.1.6"
```
Comment (dm0-) [inline app-admin/etcd-wrapper/etcd-wrapper-3.1.6.ebuild]: Since the ebuild has the same version now, maybe make this value `"v${PV}"` so updates only require a rename with no edits.

**Comment by glevand** [comment]
Used `${PV}` for etcd release tag.
Moved coreos-base/coreos changes to separate patch and bumped ebuild version.
